llama-index-integrations/indices/llama-index-indices-managed-vertexai/llama_index/indices/managed/vertexai/base.py

```diff
@@ -0,0 +1,129 @@
+"""Vertex AI Managed index.
+
+A managed Index - where the index is accessible via some API that
+interfaces a managed service.
+
+"""
+
+from typing import Any, Optional, Sequence
+
+from google.cloud.aiplatform_v1beta1 import ImportRagFilesResponse
+import vertexai
+from vertexai.preview import rag
+
+import re
+
+from llama_index.core.base.base_query_engine import BaseQueryEngine
+from llama_index.core.base.base_retriever import BaseRetriever
+from llama_index.core.indices.managed.base import BaseManagedIndex
+
+
+class VertexAIIndex(BaseManagedIndex):
```

Comment:
For now, I don't think so. This feature of Vertex AI only supports Raw files or files in GCS/Google Drive. This may change in the future to include other expansions